source/_patterns/02-organisms/22-settings/01-auto-deposit-settings.mustache

```diff
@@ -1,13 +1,26 @@
 {{#is-standalone}}
-	<form data-abide="ajax">
+	<form data-abide="ajax" class="settingsForm xbLegacyNav auto-deposit-form">
 {{/is-standalone}}
 
-<h3 class="formTitle">Auto deposits</h3>
+	<h3 class="formTitle">Monthly Good Subscription</h3>
```

Comment:
@BoulderBrains - doesn't this need to be inside a condition? (If someone has an auto-deposit but not MG, the title should still be Auto deposits.) Also, "Monthly Good subscription" should prob. be sentence case - with cap G because Good is part of name.

Reply:
Also - question for Brandon - if a user has neither auto-deposit or MG, and both prompts are showing, should the section title take in both AD and MG (e.g. "Monthly Good subscription | Auto deposit")?